pkg/ebpf/co_re.go

```diff
@@ -42,6 +42,26 @@ func LoadCOREAsset(filename string, startFn func(bytecode.AssetReader, manager.O
 	return loader.loadCOREAsset(filename, startFn)
 }
 
+// GetBTFLoaderInfo Returns where the ebpf BTF files were sourced from
+func GetBTFLoaderInfo() (string, error) {
+	loader, err := coreLoader(NewConfig())
+	if err != nil {
+		return "", err
+	}
+	res, _, err := loader.btfLoader.Get()
```

Comment:
This can cause BTF to be loaded, even if it wasn't needed before.

Reply:
You may need to add a function to `funcs.CachedFunc` to get the result without running the function.

Reply:
Hmm, also looks like `funcs.CachedFunc` doesn't deal well with the a `nil` result and `nil` error, since it is unable to distinguish that from the function not being run at all. You may need to add a `bool` to indicate the function has been run. You could add that to the new function return, something like `Result() (*T, bool, error)`

Reply:
Unfortunately it also gets removed from the cache after `btfFlushDelay` which is one minute, so it probably wouldn't be there by the time a customer runs the flare. Maybe I can instead store it as a resultMetadata field in `orderedBTFLoader` so that I don't have to re-call any of the loader functions?

Reply:
Updated this to store the data in `resultMetadata` which persists after the actual BTF files get flushed